app/scripts/views/settings/communication_preferences.js

```diff
@@ -39,22 +39,35 @@ function (Cocktail, Xss, Constants, MarketingEmailErrors, BaseView, FormView,
     beforeRender: function () {
       var self = this;
       var emailPrefs = self.getMarketingEmailPrefs();
-
       return emailPrefs.fetch()
         .fail(function (err) {
           if (MarketingEmailErrors.is(err, 'UNKNOWN_EMAIL')) {
             // user has not yet opted in to Basket yet. Ignore.
             return;
           }
+          if (MarketingEmailErrors.is(err, 'SERVICE_UNAVAILABLE')) {
+            // the service is not available, ask to retry later
+            $('.spinner').hide();
+            $('.error').html(err).show();
+            setTimeout(function() {
+              self.navigate('/');
+            }, 2000);
+            console.log(self);
+          }
```

Comment:
This makes the /lookup-user request show an error and redirect back to the accounts page. Definitely a hack, and not the right solution, but I don't know what else to do :(